client/src/components/PayPalConnect.js

```diff
@@ -1,58 +1,109 @@
 import React, { Component } from 'react';
-import { Button } from 'reactstrap';
+import { PayPalButton } from 'react-paypal-button-v2';
 import './css/PayPalConnect.css'
+import { ReactIsInDevelomentMode } from './../components/Utils';
 
 /**
  * This is the PayPal connection component for the second page of the account payment portal.
  * This component will have the user connect to paypal so we can continue with the transaction.
  * TODO: Add styling
  */
+
+const ACTIVITY_API_DEV = "http://localhost:5000/api/activity/";
+const ACTIVITY_API_PROD = "";
+
 class PayPalConnect extends Component {
     constructor(props) {
         super(props)
-        
+
         this.state = {
-            totalFee: 0.0, // TODO: replace these values with money values.
+            // TODO: get all of this information from Redux store
+            totalFee: 0.01,
             tax: 0.0,
-            paymentTotal: 0.0 // TODO: Make a state for checking to see if the user is connected to paypal already
+            paymentTotal: 0.01, 
+            householdId: '5ca780096bf2281dd453bdd8',
```

Comment:
We can actually get a real householdId and stuff now. But let's just keep this for a different PR.